### DISCUSSION TITLE
Definition of Contracts/ContractPrincipal is circular, overrestrictive and unclear

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1386] [General]
The definition is somewhat circular in that it uses "principal" to define "Principal"
Also "first"  is unclear here (earliest in time? precedence?), and it seems unnecessary given the emphasis on principal. If it's a synonym then make it such.
Finally there may be more than one principal in a contract (e.g. see [ContractThirdParty](https://spec.edmcouncil.org/fibo/ontology/FND/Agreements/Contracts/ContractThirdParty)) so "the principal" and "the first party" is too strong.

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/diffs/15_

**Comment by trypuz** [comment]
Links to concepts discussed in this issue:
- [contract principal](https://spec.edmcouncil.org/fibo/ontology/FND/Agreements/Contracts/ContractPrincipal)
- [contract third party](https://spec.edmcouncil.org/fibo/ontology/FND/Agreements/Contracts/ContractThirdParty)

BTW, we have [3 concepts labeled "principal" in FIBO now](https://spec.edmcouncil.org/fibo/ontology?searchBoxQuery=principal).

**Comment by trypuz** [comment]
@rivettp could you propose the first draft of the new definition, please?
